app/assets/javascripts/Components/dashboard.jsx

```diff
@@ -55,7 +91,9 @@ class Dashboard extends React.Component {
       return (
         <div>
           <Bar data={this.state.data} />;
-          <Bar data={this.state.data} />;
+          <Bar data={this.state.data}
+               options={this.state.options}/>;
```

Comment:
On the other hand, this line actually shouldn't be ended with a semicolon (the one on Line 93 above is a typo).